File: LES_EXAMENS/EXAM_PYTHON_2025_2026_IA_RATTRAPAGE/exerciceE.py
les_voyelles=['a','e','i','o','u','y']
chaine=str(input("saisir une chaine de caracteres: "))
taille=len(chaine)
conteur_voyelles=0
for i in range(0,taille):
    for j in range(0,6):
        if(chaine[i]==les_voyelles[j]):
            conteur_voyelles+=1
print("le nombre des voyelles est: ",conteur_voyelles);

File: LES_EXAMENS/EXAM_PYTHON_ET_EXAM_ALGO_C__2025_2026_IA_RATTRAPAGE/exerciceE.c
#include <stdio.h>
#include <string.h>
int taille,conteur_voyelles=0;
char chaine[1024];
char les_voyelles[6]={'a','e','i','o','u','y'};
int main(){
    printf("saisir une chaine de caracteres: ");
    scanf("%s",chaine);
    taille=strlen(chaine);
    for(int i=0;i<taille;i++){
        for(int j=0;j<6;j++){
            if(chaine[i]==les_voyelles[j])
            {
                conteur_voyelles++;
            }
        }
    }
    printf("le nombre des voyelles est: %d\n",conteur_voyelles);
    return 0;
}